test/subset-band-name/subset_band_name_utitlities.py

```diff
@@ -0,0 +1,42 @@
+""""
+Utility functions used by the subset-band-name regression tests.
+"""
+
+import os
+from pyhdf.SD import SD, SDC
+import numpy
+
+
+def get_sds_data(file: str):
+    sds_data = None
+    file_SD = SD(file, SDC.READ)
+    datasets = file_SD.datasets()
+
+    for key in datasets.keys():
+        if key == 'EV_250_Aggr500_RefSB':
+            print('Obtaining ', key, ' data')
+            dataset = file_SD.select(key)
+            sds_data = dataset.get()
```

Comment:
I think this could be simplified. 

```suggestion
band = 'EV_250_Aggr500_RefSB'
if band in datasets:
    print(f'Obtaining {band} data')
    sds_data = file_SD.select(band).get()
```

And then later you could add band as an input param if you were testing more than one of the variables.

Reply:
No...band will be different... It'll be subsetting the third indices for the corresponding band index numbers.

Reply:
I think @flamingbear makes a couple of great points here: it seems like this overall functionality should be useful for more than just the one collection, and therefore more than just one band name.

If the function signature was updated to `get_sds_data(file: str, band: str)`, then you could call this function to get a band for any collection.

> No...band will be different... It'll be subsetting the third indices for the corresponding band index numbers.

It doesn't look like this function is really doing anything with various indices here, just identifying things by a name.

Also, the rest of the suggestion is nice clean up - you don't need to iterate through a dictionary to find a specific key. The suggestion to just check if the key is in the dictionary (`if band in datasets`) is simpler. You do probably want to have an `else` for if you don't find the data, though - maybe an exception should be raised in such a case?